### PR TITLE
Cloud - Added change log for release 06/19

### PR DESCRIPTION
**Pull request by brayunm**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:1]: @brayunm, for the two fixes, could you rephrase the title and description so they focus on the issue rather than the solution? For fixes, it's important that a user searching for the issue can easily spot it, so we need to make sure each fix contains a clear, user-friendly description of the issue, in such a way that the user will recognize it (e.g. in case of the issue with the cookies, the user may not know that the cookies were the problem, so they may not recognize the issue from the current description).

Comment (brayunm) [reply release-notes/Cloud/DCP_change_log.md:1]: Title & descriptions have been updated

**Review by MariekeGO — APPROVED** [review]
